### PR TITLE
e2e login test

### LINKED ISSUE #51: [e2e test] Login to the SaaS starter kit UI
While running the e2e test, a user can be created on the fly and used to login to the dashboard.

### PR DESCRIPTION
**Pull request by emmacode**
e2e login test
Closes #51

### CODE REVIEW

**Comment by emmacode** [comment]
@devkiran 
Please I need your review and below is what I'm trying to do

1. In docker-compose.yml I created a test database to use for the tests
2. In logn.spec.ts when the Sign in button is clicked. It is not redirecting to the dashboard. When the button is clicked the loading is still true therefore not navigating to the dashboard. This error as well [Failed to load resource: the server responded with a status of 401 (Unauthorized)]

Screenshot of screen
[Screenshot](https://drive.google.com/drive/folders/1aCQzrELG8_eGUmFtvQGSAPeFanCt0YHi?usp=sharing)

**Comment by devkiran** [comment]
Thanks @emmacode for the PR. Let's review this and get back to you soon.

File: yarn.lock
Comment (niwsa) [inline yarn.lock:1]: Could you remove this lock file? We only maintain a single package-lock.json file.

**Review by niwsa — CHANGES_REQUESTED** [review]
@emmacode Left a few comments. Let me know in case you have any queries.

File: yarn.lock
Comment (emmacode) [reply yarn.lock:1]: yes thanks

**Comment by devkiran** [comment]
@emmacode Can you run and check if you have the same issue now? I have made some changes.

**Review by niwsa — APPROVED** [review]

**Pull request merged**